jenkins/docker/ubuntu/docs.Dockerfile

```diff
@@ -0,0 +1,71 @@
+ARG FROM_IMAGE='ubuntu:24.04'
+
+FROM ${FROM_IMAGE}
+
+# install dependencies (install tzdata first to prevent 'geographic area' prompt)
+RUN apt-get update \
+	&& apt-get install -y tzdata \
+	&& apt-get install -y openjdk-11-jdk-headless git curl libssl-dev maven ca-certificates zip unzip \
+	&& update-ca-certificates
+
+# install python
+RUN apt-get install -y python3-pip python3-venv
+
+# install shellcheck
+RUN apt-get install -y shellcheck
+
+# nodejs
+ENV NODE_OPTIONS="--dns-result-order=ipv4first"
+ARG NODEJS_VERSION=22
```

Comment:
should we use same nodejs version as javascript dockerfile?

Reply:
I will leave the same version, but they will all get bumped to next month.